From 71d4cf278c7ff0e941935bc7cefebb39d8822105 Mon Sep 17 00:00:00 2001
From: "DESKTOP-TRTPRUC\\Kiran Vemulapally" <kvemulapally@75f.io>
Date: Wed, 29 Jun 2022 17:27:02 +0530
Subject: [PATCH 05/15] 12187: latest changes; Named schedule.

---
 .../io/renatus/schedules/NamedSchedule.java   | 54 ++++++++++++-------
 1 file changed, 35 insertions(+), 19 deletions(-)

diff --git a/app/src/main/java/a75f/io/renatus/schedules/NamedSchedule.java b/app/src/main/java/a75f/io/renatus/schedules/NamedSchedule.java
index 630d83541..f8d3fdfdb 100644
--- a/app/src/main/java/a75f/io/renatus/schedules/NamedSchedule.java
+++ b/app/src/main/java/a75f/io/renatus/schedules/NamedSchedule.java
@@ -380,17 +380,25 @@ public class NamedSchedule extends DialogFragment {
             if( (double) getTuner(useCelsius.get("id").toString())== TunerConstants.USE_CELSIUS_FLAG_ENABLED) {
                  coolingDeadband = fahrenheitToCelsiusRelative(coolingDeadband);
                  heatingDeadband = fahrenheitToCelsiusRelative(heatingDeadband);
-            }
-            for (Schedule.Days namedSchedDay:namedSchedule.getDays()) {
-                double deadbandNamedSched = namedSchedDay.getCoolingVal()-namedSchedDay.getHeatingVal();
-                if(deadbandNamedSched < coolingDeadband+heatingDeadband){
-                    warningMessage.append(getText(R.string.deadband_warning)).append("\n\t").append("Deadband in Named schedule - ")
-                            .append(deadbandNamedSched).append("\n\t").append("Deadband for zone: CoolingDeadband - ").append(coolingDeadband).append("\n\t\t").append("HeatingDeadband - ").append(heatingDeadband).append("\n\n");
-                    isValid = false;
-                    break;
+                for (Schedule.Days namedSchedDay:namedSchedule.getDays()) {
+                    double deadbandNamedSched = namedSchedDay.getCoolingVal()-namedSchedDay.getHeatingVal();
+                    if(deadbandNamedSched < coolingDeadband+heatingDeadband){
+                        warningMessage.append(getText(R.string.deadband_warning)).append("\n\t").append("Deadband in Named schedule - ")
+                                .append(fahrenheitToCelsiusRelative(deadbandNamedSched)).append(" \u00B0C").append("\n\t").append("Deadband for zone: CoolingDeadband - ").append(coolingDeadband).append(" \u00B0C").append("\n\t\t").append("HeatingDeadband - ").append(heatingDeadband).append(" \u00B0C").append("\n\n");
+                        isValid = false;
+                        break;
+                    }
+                }
+            } else {
+                for (Schedule.Days namedSchedDay : namedSchedule.getDays()) {
+                    double deadbandNamedSched = namedSchedDay.getCoolingVal() - namedSchedDay.getHeatingVal();
+                    if (deadbandNamedSched < coolingDeadband + heatingDeadband) {
+                        warningMessage.append(getText(R.string.deadband_warning)).append("\n\t").append("Deadband in Named schedule - ")
+                                .append(deadbandNamedSched).append(" \u00B0F").append("\n\t").append("Deadband for zone: CoolingDeadband - ").append(coolingDeadband).append("\u00B0F").append("\n\t\t").append("HeatingDeadband - ").append(heatingDeadband).append("\u00B0F").append("\n\n");
+                        isValid = false;
+                        break;
+                    }
                 }
-
-
             }
 
             StringBuilder desiredTempWarning = new StringBuilder();
@@ -406,14 +414,14 @@ public class NamedSchedule extends DialogFragment {
                         desiredTempWarning.append("\t\t").append(dayName[namedSchedDay.getDay()]).append("-")
                                 .append(namedSchedDay.getSthh()).append(":").append(namedSchedDay.getStmm())
                                 .append("-").append(namedSchedDay.getEthh()).append(":").append(namedSchedDay.getEtmm())
-                                .append("(CDT - ").append(fahrenheitToCelsius(namedSchedDay.getCoolingVal())).append(";")
-                                .append("HDT - ").append(fahrenheitToCelsius(namedSchedDay.getHeatingVal())).append(")").append("\n\t\t");
+                                .append("(CDT - ").append(fahrenheitToCelsius(namedSchedDay.getCoolingVal())).append(" \u00B0C").append(";")
+                                .append("HDT - ").append(fahrenheitToCelsius(namedSchedDay.getHeatingVal())).append(" \u00B0C").append(")").append("\n\t\t");
                     } else {
                         desiredTempWarning.append("\t\t").append(dayName[namedSchedDay.getDay()]).append("-")
                                 .append(namedSchedDay.getSthh()).append(":").append(namedSchedDay.getStmm())
                                 .append("-").append(namedSchedDay.getEthh()).append(":").append(namedSchedDay.getEtmm())
-                                .append("(CDT - ").append(namedSchedDay.getCoolingVal()).append(";")
-                                .append("HDT - ").append(namedSchedDay.getHeatingVal()).append(")").append("\n\t\t");
+                                .append("(CDT - ").append(namedSchedDay.getCoolingVal()).append(" \u00B0F").append(";")
+                                .append("HDT - ").append(namedSchedDay.getHeatingVal()).append(" \u00B0F").append(")").append("\n\t\t");
                     }
                     isValid = false;
                     isDesiredTempValid= false;
@@ -425,15 +433,15 @@ public class NamedSchedule extends DialogFragment {
                     warningMessage.append(getText(R.string.desiredTemp_warning)).append("\n\t")
                             .append("Named schedule desired temperature : \n\t\t").append(desiredTempWarning)
                             .append("Building desired temps limit : HDT - ")
-                            .append((fahrenheitToCelsius(buildingTuner.getMinHeatingUserLimit()))).append("~").append((fahrenheitToCelsius(buildingTuner.getMaxHeatingUserLimit())))
-                            .append("\n\t\t CDT - ").append(fahrenheitToCelsius(buildingTuner.getMinCoolingUserLimit())).append("~").append(fahrenheitToCelsius(buildingTuner.getMaxCoolingUserLimit()))
+                            .append((fahrenheitToCelsius(buildingTuner.getMinHeatingUserLimit()))).append(" \u00B0C").append("~").append((fahrenheitToCelsius(buildingTuner.getMaxHeatingUserLimit()))).append(" \u00B0C")
+                            .append("\n\t\t CDT - ").append(fahrenheitToCelsius(buildingTuner.getMinCoolingUserLimit())).append(" \u00B0C").append("~").append(fahrenheitToCelsius(buildingTuner.getMaxCoolingUserLimit())).append(" \u00B0C")
                             .append("\n\n");
                 } else {
                     warningMessage.append(getText(R.string.desiredTemp_warning)).append("\n\t")
                             .append("Named schedule desired temperature : \n\t\t").append(desiredTempWarning)
                             .append("Building desired temps limit : HDT - ")
-                            .append(buildingTuner.getMinHeatingUserLimit()).append("~").append(buildingTuner.getMaxHeatingUserLimit())
-                            .append("\n\t\t CDT - ").append(buildingTuner.getMinCoolingUserLimit()).append("~").append(buildingTuner.getMaxCoolingUserLimit())
+                            .append(buildingTuner.getMinHeatingUserLimit()).append(" \u00B0F").append("~").append(buildingTuner.getMaxHeatingUserLimit()).append(" \u00B0F")
+                            .append("\n\t\t CDT - ").append(buildingTuner.getMinCoolingUserLimit()).append(" \u00B0F").append("~").append(buildingTuner.getMaxCoolingUserLimit()).append(" \u00B0F")
                             .append("\n\n");
                 }
             }
@@ -541,7 +549,15 @@ public class NamedSchedule extends DialogFragment {
         if(getContext()==null) return;
         AppCompatTextView textViewTemp = new AppCompatTextView(getContext());
         textViewTemp.setGravity(Gravity.CENTER_HORIZONTAL);
-        textViewTemp.setText(Html.fromHtml(strminTemp + " " + strmaxTemp,Html.FROM_HTML_MODE_LEGACY));
+        String celsiusUnitMin = FontManager.getColoredSpanned("\u00B0C", colorMinTemp);
+        String celsiusUnitMax = FontManager.getColoredSpanned("\u00B0C", colorMaxTemp);
+        String farenUnitMin = FontManager.getColoredSpanned("\u00B0F", colorMinTemp);
+        String farenUnitMax = FontManager.getColoredSpanned("\u00B0F", colorMaxTemp);
+        if( (double) getTuner(useCelsius.get("id").toString())== TunerConstants.USE_CELSIUS_FLAG_ENABLED) {
+            textViewTemp.setText(Html.fromHtml(strminTemp + celsiusUnitMin + " " + strmaxTemp + celsiusUnitMax, Html.FROM_HTML_MODE_LEGACY));
+        } else {
+            textViewTemp.setText(Html.fromHtml(strminTemp + farenUnitMin + " " + strmaxTemp + farenUnitMax, Html.FROM_HTML_MODE_LEGACY));
+        }
         if(typeface!=null)
             textViewTemp.setTypeface(typeface);
         TextViewCompat.setAutoSizeTextTypeWithDefaults(textViewTemp, TextViewCompat.AUTO_SIZE_TEXT_TYPE_UNIFORM);
-- 
2.31.0.windows.1

