From 35b1621f53e63127303e353efa6ef79d6d8c7963 Mon Sep 17 00:00:00 2001
From: 75fdevelopment <75fdevelopment@75f.io>
Date: Thu, 16 Jun 2022 14:45:17 +0000
Subject: [PATCH 07/35] Version bump to 1.622.0

---
 CHANGES           | 170 +++++++++++++++++++++++++++++++++++++++++-----
 VERSION           |   2 +-
 gradle.properties |   2 +-
 3 files changed, 155 insertions(+), 19 deletions(-)

diff --git a/CHANGES b/CHANGES
index 24a82830e..4808363d6 100644
--- a/CHANGES
+++ b/CHANGES
@@ -1,17 +1,153 @@
- - Merged PR 4164: Update mechanical cooling when lockout disabled
- - Merged PR 4054: CCU : Named Schedule : Desired Temp Limits Validation
- - Merged PR 4149: Use system conditioning while applying damper limits config
- - Merged PR 4092: 12203
- - Use system conditioning while applying damper limits config
- - Merged PR 4055: Hyperstat Desired temp validation
- - Removed commented code
- - Removed commented code
- - Merge branch 'master' of https://dev.azure.com/75fdevelopment/75F%20Engineering/_git/ccu into userstory/11485_Local_Control_settings_on_ITM_are_ignored_on_Renatus
- - Merged PR 4097: DAB invalid system status and normalization fix
- - Merged PR 3962: Randomize CCU 15 minutes his write
- - Update mechanical cooling when lockout disabled
- - DAB invalid system status and normalization fix
- - Fixed some sonar cure changes
- - Added Unit test cases for desired temp validation functions
- - validating the single desired temperature point. If valid desired temp considering it. If invalid then setting to the nearest boundary limit and updating to desired temp
- - Randomize CCU 15 minutes his write
\ No newline at end of file
+ - Merged PR 4251: True CFM Implementation for DAB
+ - Merged PR 4114: TrueCFM dab algorithm implementation
+ - Merged PR 4160: Mapping local oat to outsideTemp
+ - Merged PR 4161: History sync issue
+ - merge master
+ - Merged PR 4224: Restoring Diag points
+ - Merge branch 'master' of https://dev.azure.com/75fdevelopment/75F%20Engineering/_git/ccu into bug/mkundaragi/12446_Bug_12446_CCU_Not_Reporting_Proper_App_Version_in_Portal
+ - Resolved issue while add new ccu for existing site also
+ - Merged PR 4091: Dis name for analog input changed for piloop and sense.
+ - Merge branch 'master' of https://dev.azure.com/75fdevelopment/75F%20Engineering/_git/ccu into bug/mkundaragi/12446_Bug_12446_CCU_Not_Reporting_Proper_App_Version_in_Portal
+ - Merged PR 4135: 12285 : CCU | BPOS| zone is not reverting to temporary hold (auto)
+ - Merged PR 4042: Bug 11411: CCU : Overlapping in building schedule after editing zone schedule
+ - Merged PR 4109: Bug 11758: CCU: Changing floor name disrupts zone modbus communication
+ - Merged PR 3715: TrueCfm algorithm implementation
+ - Merge master
+ - fix sonarqube warnings
+ - Merge branch 'master' of https://dev.azure.com/75fdevelopment/75F%20Engineering/_git/ccu into bug/mkundaragi/12446_Bug_12446_CCU_Not_Reporting_Proper_App_Version_in_Portal
+ - Merged PR 4110: Alert not resolving
+ - resolved merge conflicts
+ - Merge branch 'master' of https://dev.azure.com/75fdevelopment/75F%20Engineering/_git/ccu into bug/mkundaragi/12446_Bug_12446_CCU_Not_Reporting_Proper_App_Version_in_Portal
+ - Merged PR 4141: VOC and pm2.5 configuration for HS & minor UI changes
+ - handle deadband
+ - Added Daig migration 1. Checking the diag points has  gateway reff if no adding to diag eqip 2. If there is no local diag points, than fetching from silo
+ - Added Daig migration 1. Checking the diag points has  gateway reff if no adding to diag eqip 2. If there is no local diag points, than fetching from silo
+ - Added Daig migration 1. Checking the diag points has  gateway reff if no adding to diag eqip 2. If there is no local diag points, than fetching from silo
+ - Migration draft
+ - Added migration for missing Daig Points
+ - Merge branch 'master' of https://dev.azure.com/75fdevelopment/75F%20Engineering/_git/ccu into bug/mkundaragi/12446_Bug_12446_CCU_Not_Reporting_Proper_App_Version_in_Portal
+ - Merged PR 4136: Vacation bug
+ - Merge branch 'master' of https://dev.azure.com/75fdevelopment/75F%20Engineering/_git/ccu into bug/mkundaragi/12446_Bug_12446_CCU_Not_Reporting_Proper_App_Version_in_Portal
+ - Merged PR 4153: Added new Enum for FirmwareComponentType_t
+ - Merge branch 'master' of https://dev.azure.com/75fdevelopment/75F%20Engineering/_git/ccu into bug/mkundaragi/12446_Bug_12446_CCU_Not_Reporting_Proper_App_Version_in_Portal
+ - Changed the default system profile daig point creation sequence
+ - Merged PR 4009: 11565 : CCU screen should fall back to Zone screen
+ - Fix vav query
+ - crash resolved
+ - Merge branch 'bug/sdev/12203_delete_schedule' of https://dev.azure.com/75fdevelopment/75F%20Engineering/_git/ccu into bug/bharath/11411_overlapping_schedules
+ - Merge branch 'master' of https://dev.azure.com/75fdevelopment/75F%20Engineering/_git/ccu into bug/sdev/12203_delete_schedule
+ - update push
+ - try catch added
+ - Merge branch 'master' of https://dev.azure.com/75fdevelopment/75F%20Engineering/_git/ccu into bug/bharath/11411_overlapping_schedules
+ - Corner cases
+ - History sync issue
+ - crash resolved
+ - Merge branch 'master' of https://dev.azure.com/75fdevelopment/75F%20Engineering/_git/ccu into bug/sdev/12203_delete_schedule
+ - Merge branch 'master' of https://dev.azure.com/75fdevelopment/75F%20Engineering/_git/ccu into bug/bharath/11411_overlapping_schedules
+ - unwanted changes removed
+ - Bug 11758: CCU: Changing floor name disrupts zone modbus communication
+ - Bug 11758: CCU: Changing floor name disrupts zone modbus communication
+ - Merge master
+ - Merge branch 'master' of https://dev.azure.com/75fdevelopment/75F%20Engineering/_git/ccu into bug/sdev/12203_delete_schedule
+ - Overlapping issue resolved
+ - Merge branch 'master' of https://dev.azure.com/75fdevelopment/75F%20Engineering/_git/ccu into bug/bharath/11411_overlapping_schedules
+ - Mapping local oat to outsideTemp
+ - Merge branch 'Story/Amaresh/True_cfm_vav_latest' of https://dev.azure.com/75fdevelopment/75F%20Engineering/_git/ccu into story/samjith/5325_truecfm_vav_algorithm
+ - Merge master
+ - crash on deletion fixed
+ - Merge master
+ - Merge branch 'master' into story/samjith/5325_truecfm_vav_algorithm
+ - Merge branch 'Story/Amaresh/True_Cfm_Dab_Latest' of https://dev.azure.com/75fdevelopment/75F%20Engineering/_git/ccu into story/samjith/5326_truecfm_dab_algo
+ - default value for minCfm is changed back to 100
+ - comment resolved
+ - sonarQ issue resolved
+ - Bug 11411: CCU : Overlapping in building schedule after editing zone schedule
+ - Handle damper when truecfm disabled
+ - Revert "Handled migration for pressurePoint"
+ - Merge branch 'story/samjith/5326_truecfm_dab_algo' of https://dev.azure.com/75fdevelopment/75F%20Engineering/_git/ccu into Story/Amaresh/True_Cfm_Dab_Latest
+ - Merge branch 'master' into story/samjith/5326_truecfm_dab_algo
+ - refactor code
+ - changed default value for mincfmiaq from 100 to 900
+ - Merge branch 'Story/Amaresh/True_Cfm_Dab_Latest' of https://dev.azure.com/75fdevelopment/75F%20Engineering/_git/ccu into story/Amaresh/True_Cfm_Dab_Latest
+ - Added unit for minCFMIAQ
+ - Merge branch 'Story/Amaresh/True_Cfm_Dab_Latest' of https://dev.azure.com/75fdevelopment/75F%20Engineering/_git/ccu into story/samjith/5326_truecfm_dab_algo
+ - TrueCfm DAB algorithm- initial implementation
+ - Reconfiguration support for DAB
+ - Merge branch 'Story/Amaresh/True_cfm_vav_latest' of https://dev.azure.com/75fdevelopment/75F%20Engineering/_git/ccu into Story/Amaresh/True_Cfm_Dab_Latest
+ - Merge branch 'master' of https://dev.azure.com/75fdevelopment/75F%20Engineering/_git/ccu into Story/Amaresh/True_cfm_vav_latest
+ - Handled migration for pressurePoint
+ - resolved multiple sp points creation issue
+ - fixed observation in reconfiguration support-4 "if maxCfm is  reduced less than minCfm from portal, then max value of minCfm should be less <= maxCfm"
+ - Merge branch 'Story/Amaresh/True_Cfm_Dab_Latest' of https://dev.azure.com/75fdevelopment/75F%20Engineering/_git/ccu into story/samjith/5326_truecfm_dab_algo
+ - Merge branch 'master' of https://dev.azure.com/75fdevelopment/75F%20Engineering/_git/ccu into Story/Amaresh/True_cfm_vav_latest
+ - fixed observation in reconfiguration support-3
+ - fixed observation in reconfiguration support-2
+ - fixed observation in reconfiguration support
+ - Merge branch 'master' of https://dev.azure.com/75fdevelopment/75F%20Engineering/_git/ccu into Story/Amaresh/True_cfm_vav_latest
+ - updated changes
+ - created common method for adding iaq point
+ - True CFM DAB UI and model implementation Reconfiguration and migration support
+ - Merge master
+ - Add logs and damper limit check
+ - Merge branch 'Story/Amaresh/True_cfm_vav_latest' of https://dev.azure.com/75fdevelopment/75F%20Engineering/_git/ccu into story/Amaresh/True_Cfm_Dab_Latest
+ - Merge branch 'Story/Amaresh/True_cfm_vav_latest' of https://dev.azure.com/75fdevelopment/75F%20Engineering/_git/ccu into story/samjith/5325_truecfm_vav_algorithm
+ - Initialize cfm loop in series and parallel fan profiles
+ - fix for observation3
+ - repush
+ - Airflow cfm becomes invisible in zones page when cfm is disabled
+ - fix for improper tags to read airflow cfm and default value for min cfm cooling abd reheating
+ - updated test cases
+ - updated reconfiguration for vav-2
+ - updated reconfiguration for vav
+ - Merge branch 'story/samjith/5325_truecfm_vav_algorithm' of https://dev.azure.com/75fdevelopment/75F%20Engineering/_git/ccu into Story/Amaresh/True_cfm_vav_latest
+ - Added min and max tags
+ - Fix enable tag
+ - merge
+ - Resolved merge conflicts
+ - Merge branch 'story/samjith/5325_truecfm_vav_algorithm' of https://dev.azure.com/75fdevelopment/75F%20Engineering/_git/ccu into Story/Amaresh/True_cfm_vav_latest
+ - Merge branch 'Story/Amaresh/truecfm_vav_ui_model' of https://dev.azure.com/75fdevelopment/75F%20Engineering/_git/ccu into Story/Amaresh/Truecfmvav5323
+ - merge master
+ - changed tag type enable to enabled
+ - Added enums and control point
+ - Add cfm units and fix enable tag
+ - updated tags and resolved conflicts
+ - Merge branch 'story/samjith/5325_truecfm_vav_algorithm' of https://dev.azure.com/75fdevelopment/75F%20Engineering/_git/ccu into story/Amaresh/truecfm_vav_ui_model
+ - Tag changes
+ - Merge branch 'story/Amaresh/truecfm_vav_ui_model' into story/samjith/5325_truecfm_vav_algorithm
+ - Update tuner points
+ - Merge UI branch
+ - Merge branch 'master' into story/samjith/5325_truecfm_vav_algorithm
+ - Resolved merge conflicts
+ - Resolved merge conflicts
+ - Resolved merge conflicts
+ - Revert "Merged PR 3588: Airflow volume unit migration"
+ - Merge branch 'master' of https://dev.azure.com/75fdevelopment/75F%20Engineering/_git/ccu into Story/Amaresh/truecfm_vav_ui_model
+ - Fixes for app crashing while Default Tuner creation and while Migration
+ - Merge branch 'Story/Amaresh/truecfm_vav_ui_model' of https://dev.azure.com/75fdevelopment/75F%20Engineering/_git/ccu into Story/Amaresh/truecfm_vav_ui_model
+ - Merged PR 3688: TrueCFM branch merge
+ - Resolve merge conflicts
+ - Merge branch 'master' of https://dev.azure.com/75fdevelopment/75F%20Engineering/_git/ccu into Story/Amaresh/truecfm_vav_ui_model
+ -  Reconfiguration support for TrueCFM -VAV
+ - Fix invalid damper when system is cooling and zone is heating
+ - cfm algorithm implementation
+ - Resolved merge conflicts
+ - Merge branch 'master' of https://dev.azure.com/75fdevelopment/75F%20Engineering/_git/ccu into Story/Amaresh/truecfm_vav_ui_model
+ -  True cfm Migration support for vav
+ - Merge branch 'story/Amaresh/truecfm_vav_ui_model' into story/samjith/5325_truecfm_vav_algorithm
+ - Remove duplicate tuner creation
+ - Merge branch 'story/Amaresh/truecfm_vav_ui_model' into story/samjith/5325_truecfm_vav_algorithm
+ - Fix misc issues with point creation
+ - Merge branch 'story/Amaresh/truecfm_vav_ui_model' into story/samjith/5325_truecfm_vav_algorithm
+ - handle creation and deletion of damper config points
+ - Initial implementation of VAV - truecfm
+ - Added fanMarker tag to new config points
+ - Revert "created static variables in VavProfileConfiguration class"
+ - created static variables in VavProfileConfiguration class
+ - sonar cube issues resolved
+ - updated Test cases
+ - updated changes for vavTunerPoints
+ - refactor forcepush
+ - Refactor config point class
+ - Created common file for TrueCFMConfig points so it can be reused for DAB profile
+ - sonar cube issues resolved
+ - True CFM VAV UI and model Implementation
\ No newline at end of file
diff --git a/VERSION b/VERSION
index 5f7fef44d..d1ebdddca 100644
--- a/VERSION
+++ b/VERSION
@@ -1 +1 @@
-1.621.0
+1.622.0
diff --git a/gradle.properties b/gradle.properties
index 362cece89..8365b5913 100644
--- a/gradle.properties
+++ b/gradle.properties
@@ -20,7 +20,7 @@ ANDROID_BUILD_MIN_SDK_VERSION = 19
 ANDROID_BUILD_TARGET_SDK_VERSION = 19
 android.enableD8=true
 
-VERSION_CODE_MINOR=621
+VERSION_CODE_MINOR=622
 VERSION_CODE_MAJOR=1
 VERSION_CODE_PATCH=0
 org.gradle.jvmargs=-Xmx4608M
-- 
2.31.0.windows.1

